src/components/add-new-item/add-new-item.js

```diff
@@ -1,50 +1,46 @@
-import { Component } from "react";
+import { useState } from "react";
 
-export default class AddNewItem extends Component {
-  state = {
-    newTask: "",
-  };
+const AddNewItem = ({
+  prepareToAddNewItem,
+  addNewItem,
+  isHidden,
+  onCancelPress,
```

Comment:
check on the article how to name functions in react
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
For now I just change architecture, so it can be fixed in next branches. Is it will be ok?